app/views/jokes.py

```diff
@@ -34,3 +35,39 @@ def joke(joke_id):
     current_joke = Joke.query.filter_by(id=joke_id).first()
     return render_template('jokes/joke.html',
                            joke=current_joke)
+
+@JOKES_MOD.route('/rate-joke', methods=['POST'])
+def rate_joke():
+    """add new reaction to database"""
+    joke_id = request.form["joke_id"]
+    reaction_id = request.form["reaction_id"]
+    try:
+        reaction_id = int(reaction_id)
+    except:
+        return False
+
+    try:
+        joke_id = int(joke_id)
+    except:
+        return False
+
+    if reaction_id == 1:
```

Comment:
no need for the if-ladder, enum items can be accessed by their value like:
reaction = ReactionsType(reaction_id)
(you can put a try-except around it to catch invalid values)